packages-next/fields-document/src/tests/test-fixtures.ts

```diff
@@ -15,10 +14,8 @@ export const subfieldName = 'document';
 
 export const fieldConfig = () => ({ ___validateAndNormalize: (x: any) => x });
```

Comment:
I'm assuming fieldConfig gets passed to `document`?

```suggestion
export const fieldConfig = () => ({});
```